### PR TITLE
Add intaller e2e tests for oVirt platform

### PR DESCRIPTION
**Pull request by rgolangh**
This PR will allow running e2e of openshift/installer for ovirt.
oVirt platform installer patch is still in review - https://github.com/openshift/installer/pull/1948
 
Related: github.com/openshift/installer/pull/1948
Enhancement: https://github.com/openshift/enhancements/pull/61

### CODE REVIEW

File: cluster/ci/config/prow/plugins.yaml [hunk 1]
```diff
@@ -1155,6 +1155,10 @@ config_updater:
       name: prow-job-cluster-launch-installer-openstack-e2e
       additional_namespaces:
       - ci-stg
+    ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml:
```
Comment (stevekuznetsov) [inline cluster/ci/config/prow/plugins.yaml]: Can you propose the changes to this file in a separate PR to merge first?

Comment (rgolangh) [reply cluster/ci/config/prow/plugins.yaml]: Moved to this new PR #4378

**Comment by wking** [comment]
> Why does this need to be a separate template?

Do we?  I'd much prefer conditionals in existing templates.

File: ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml [hunk 2]
```diff
@@ -1249,6 +1249,63 @@ presubmits:
           name: prow-job-cluster-launch-installer-openstack-e2e
         name: job-definition
     trigger: (?m)^/test( | .* )e2e-openstack-parallel,?($|\s.*)
+  - agent: kubernetes
```
Comment (stevekuznetsov) [inline ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml]: You still need to label this job ...

```
    labels:
      pj-rehearse.openshift.io/can-be-rehearsed: "true"
```

Comment (rgolangh) [reply ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml]: done

**Comment by stevekuznetsov** [comment]
v

File: Makefile [hunk 3]
```diff
@@ -43,10 +43,10 @@ services:
 
 # these are useful for devs
 jobs:
-	docker pull registry.svc.ci.openshift.org/ci/ci-operator-prowgen:latest
-	docker run -it -v "${CURDIR}/ci-operator:/ci-operator" registry.svc.ci.openshift.org/ci/ci-operator-prowgen:latest --from-dir /ci-operator/config --to-dir /ci-operator/jobs
-	docker pull registry.svc.ci.openshift.org/ci/determinize-prow-jobs:latest
-	docker run -it -v "${CURDIR}/ci-operator/jobs:/ci-operator/jobs" registry.svc.ci.openshift.org/ci/determinize-prow-jobs:latest --prow-jobs-dir /ci-operator/jobs
+	podman pull registry.svc.ci.openshift.org/ci/ci-operator-prowgen:latest
```
Comment (stevekuznetsov) [inline Makefile]: you should alias `docker` to `podman` on your machine -- this is a no-go for merging into this repo

**Comment by petr-muller** [comment]
@rgolangh I think this needs a review and approval by by someone who knows templates:

https://github.com/openshift/release/blob/master/ci-operator/templates/openshift/installer/OWNERS

I'll happily approve the `jobs` and `secret-mirroring ` bits afterwards, they look correct to me.

File: ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml [hunk 4]
```diff
@@ -0,0 +1,468 @@
...
+  required: true
+- name: CLUSTER_TYPE
+  value: ovirt
+- name: TEST_COMMAND
+  required: true
+- name: RELEASE_IMAGE_LATEST
+  required: true
+- name: BASE_DOMAIN
+  required: true
+  value: gcp.devcluster.openshift.com
```
Comment (rgolangh) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml]: We have a duplicated `required` key here, I'll push a fix for that

**Comment by enxebre** [comment]
Can we please make sure this template runs `must-gather`

**Pull request merged**